code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -380,7 +380,7 @@ var/global/list/animal_count = list() //Stores types, and amount of animals of t
 /mob/living/simple_animal/bullet_act(var/obj/item/projectile/Proj)
 	if(!Proj)
 		return
-	// FUCK mice. - N3X
+	// mice are pretty cool guys
```

Comment:
why are we not just overriding bullet act for mice?? typechecking src is bad and this returns after anyway

Reply:
I was just changing N3X15's comment trash talking mice, but that's a good catch. I'll throw in a fix moving that bit down to `/mouse`.